.env

```diff
@@ -0,0 +1 @@
+TG_TOKEN=369433381:AAF4mnXcr1Zp_KQ_5So5BZ2B1pLxmD5yhCw
```

Comment:
and .env to .gitignore
and .env.example for everyone who wants to use your bot.